README.md

```diff
@@ -396,9 +364,106 @@ Finally, add the `App__peers-count` CSS class to the bottom of `App.css`:
 
 You should now be able to see the number of peers collaborating on the To-dos. Depending on the network, it might take some time to discover peers.
 
+### 7. Testing if the application works with other users
+
+Open the application in two different browsers, e.g.: Chrome and Chrome incognito. Any changes should replicate seamlessly. Be sure to also make changes while offline and see if they syncronize correctly once online.
```

Comment:
"make changes while offline and see if they syncronize correctly once online" this won't really if you're using two browsers locally and disconnect your wifi, they should still be connected, correct? (unless, you're using ws-star I guess...)

Reply:
Good question. I've tried with Chrome and Safari and they lost connection when WIFI is turned off. @pgte ?

Reply:
They will be disconnected because they're using a public ws-star server, where all the connection are piped through.. Would be true if the transport was WebRTC.
BTW, I've opened [this issue ](https://github.com/ipfs-shipyard/peer-star-app/issues/42) to try migrating to WebRTC.